binderhub/main.py

```diff
@@ -46,6 +53,7 @@ def get(self, provider_prefix, _unescaped_spec):
             raise HTTPError(400, str(e))
 
         provider_spec = f'{provider_prefix}/{spec}'
+        social_desc = f"{SPEC_NAMES[provider_prefix]}: {spec}"
```

Comment:
I'll merge this now but I think we need to chop off the first part of the `spec` so that we get "GitHub repository: <org>/<repo>" instead of what the current code will give us: "GitHub repository: gh/<org>/<repo>". At least that is what my neural network python parser says ;)

Reply:
☝️ @choldgraf

Reply:
I don't *think* that's true (at least, I think that the `provider_spec` is the thing that includes `gh`, while `spec` should just be the stuff after, but I just deployed it to mybinder.org and let's see :-)

Reply:
From what we see on twitter: you were right :D